src/__experimental__/components/switch/switch.component.js

```diff
@@ -135,7 +135,7 @@ Switch.propTypes = {
    * Set the size of the Switch to 'small' (16x16 - default) or 'large' (24x24).
    * No effect when using Classic theme.
```

Comment:
We don't have `classic theme` anymore. Is it worth changing this comment?

Reply:
I think that could be addressed later in a separate commit, as the Classic Theme is referenced at least five times in our code